src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -251,6 +251,18 @@ private function addFirewallsSection(ArrayNodeDefinition $rootNode, array $facto
                     ->scalarNode('path')->defaultValue('/logout')->end()
                     ->scalarNode('target')->defaultValue('/')->end()
                     ->booleanNode('invalidate_session')->defaultTrue()->end()
+                    ->scalarNode('clear_site_data')->defaultValue('off')->end()
+                    ->arrayNode('clear_site_data')
+                        ->fixXmlConfig('clear_site_data')
```

Comment:
This call is on the wrong place. It must go on the parent node (the one that can fix the name of its children)